module-apps/application-messages/ApplicationMessages.cpp

```diff
@@ -70,6 +72,16 @@ namespace app
             }
             return sys::MessageNone{};
         });
+
+        connect(typeid(locks::UnlockedPhone), [&](sys::Message *msg) -> sys::MessagePointer {
```

Comment:
This could be done as one type, e.g. `PhoneLockStateMessage`, carrying a `PhoneLockState` enum value.

Reply:
Message is not introduced in this PR and also is used in another services.